### ISSUE TITLE
Seeders not being created

### ISSUE THREAD

**Issue by coldKingdom** [laravel-shift/blueprint#216]
- Laravel Version: 7.11.0
- PHP Version: 7.4.5
- Blueprint Version: 1.11.0
- Platform: Windows | Linux (WSL)

### Description:
The seeders are not being created when running php artisan blueprint:build .

### Steps To Reproduce:

I created this draft.yaml
```yaml
models:
  User:
    name: string
    email: string unique
    email_verified_at: datetime nullable
    password: string
    remember_token: remembertoken
  Images:
    title: string:200
    content: text
    user_id: foreign:id unsigned
  Journeys:
    title: string:400
    content: text
    user_id: foreign:id unsigned
    published_at: datetime nullable
  News:
    title: string:400
    content: text
    user_id: foreign:id unsigned

seeders: User, Images, Journeys, News
```

When running php artisan blueprint:build the seeders is being ignored.
Only the models, migration and factories are being created.

**Comment by jasonmccreary** [comment]
I see the issue. I'll get this fixed and patched up shortly.

**Comment by jasonmccreary** [comment]
This should be fixed in the latest patch release - 1.11.1.

Run `composer update` to get the latest.
